docs/paper/admin/reference/configuration/global-configuration.mdx

```diff
@@ -0,0 +1,415 @@
+---
+slug: /reference/global-configuration
+---
+
+# Global Configuration
+
+import React from 'react';
+import Config from '@site/src/components/Config';
+
+:::info
+
+The below YAML shows you the structure and default values for the global configuration.
+
+Click on a leaf node to view the description for that setting.
+
+:::
+
+<Config
+    data={`
+block-updates:
+  disable-noteblock-updates:
+    default: 'false'
+    description: 'Whether to disable any form of block updates for note blocks on
+      the server. Disabling block updates leads to note blocks no longer updating
+      their block state, allowing for technically invalid note blocks to remain in
+      the world, which might be useful for mapmakers.'
+  disable-tripwire-updates:
+    default: 'false'
+    description: 'Whether to disable any form of block updates for tripwires on the
+      server. Disabling block updates leads to tripwires no longer updating their
+      block state, allowing for technically invalid tripwires to remain in the world,
+      which might be useful for mapmakers.'
+chunk-loading:
```

Comment:
I had struggle to found why this section wasn't implemented in the server anymore before i found it (https://github.com/PaperMC/Paper/blob/master/patches/server/0019-Rewrite-chunk-system.patch#L15697, this patch doesn't remove the section in the proper patch, nor remove/upgrade the outdated config). So in short you can drop this whole section and someone need to update the patch.

Reply:
Yeah, it seems like this was dropped in favour of `chunk-loading-basic` and `chunk-loading-advanced`. I have addressed all of your comments now. Thanks for going through this all with a fine toothpick.